src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -6544,7 +6544,7 @@ Real64 SurroundingSurfacesRadCoeffAverage(EnergyPlusData &state, int const SurfN
     // compute exterior surfaces LW radiation transfer coefficient to surrounding surfaces
     // the surface.SrdSurfTemp is weighed by surrounding surfaces view factor
     Real64 HSrdSurf = 0.0;
-    auto &surface = state.dataSurface->Surface(SurfNum);
+    auto const &surface = state.dataSurface->Surface(SurfNum);
     Real64 SrdSurfsTK = surface.SrdSurfTemp + Constant::Kelvin;
```

Comment:
[src/EnergyPlus/ConvectionCoefficients.cc:4691]:(style),[constVariable],Variable 'surfIntConv' can be declared as reference to const
[src/EnergyPlus/ConvectionCoefficients.cc:6547]:(style),[constVariable],Variable 'surface' can be declared as reference to const